user-interface/src/main/java/life/qbic/datamanager/parser/xlsx/XLSXParser.java

```diff
@@ -0,0 +1,109 @@
+package life.qbic.datamanager.parser.xlsx;
+
+import static java.util.Objects.isNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+import life.qbic.datamanager.parser.MetadataParser;
+import life.qbic.datamanager.parser.ParsingResult;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFRow;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+/**
+ * <b>XLSX parser implementation of the {@link MetadataParser} interface.</b>
+ * <p>
+ * Parses information from content following the XLSX format specification:
+ * <p>
+ * <a
+ * href="https://learn.microsoft.com/en-us/openspecs/office_standards/ms-xlsx/2c5dee00-eff2-4b22-92b6-0738acd4475e">
+ * https://learn.microsoft.com/en-us/openspecs/office_standards/ms-xlsx/2c5dee00-eff2-4b22-92b6-0738acd4475e</a>
+ *
+ * @since 1.4.0
+ */
+public class XLSXParser implements MetadataParser {
+
+  private boolean headerToLowerCase = false;
+
+  private XLSXParser() {
+  }
+
+  private XLSXParser(boolean headerToLowerCase) {
+    this.headerToLowerCase = headerToLowerCase;
+  }
+
+  public static XLSXParser create() {
+    return new XLSXParser(false);
+  }
+
+  public static XLSXParser createWithHeaderToLowerCase() {
+    return new XLSXParser(true);
+  }
+
+  private static String readCellAsString(Cell cell) {
+    return switch (cell.getCellType()) {
+      case _NONE, ERROR, BOOLEAN, FORMULA -> null;
```

Comment:
Shouldn't Boolean lead to `true` and `false` rather than null?

Reply:
Sorry, not sure what you mean :/

Reply:
For `BOOLEAN` fields we could use `Boolean.parseBoolean(String input)`. Und `Boolean.toString(boolean value)`